src/client/containers/LandingPage/LandingPage.Container.js

```diff
@@ -29,6 +29,7 @@ const LandingPageContainer = () => {
         ) : (
           <Carousel
             imageArray={imageArray.map((product) => product.picture)}
+            products={products.data}
```

Comment:
Since product.data already contains images.  I suggest you pass only it and map images inside Carousel. It will be more logical. Also you can add relevant alt to all images from product.name. (this will affect product page also since it is using carousel)

Reply:
I thought this but I don't want to change too many things. Because it is affecting product page too. But yes maybe I can make some changes to these 3 files

Reply:
I did some changes but it is changing too many codes and I am getting storybook error with carousel. Maybe I can turn back my old commit.